docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -366,6 +371,14 @@ Incorrectly formatted override files may be accepted without warnings or errors
       * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
       * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
 
+* If the Administration Server pod do start but fails to reach ready state:
+  * Check if there is a message ` WebLogic server failed to start due to missing or invalid situational configuration files` in the Administration Server pod's `kubectl log`
```

Comment:
Check if there is a message -> Check for this message

Reply:
Fixed